### ISSUE TITLE
Need a callback function for the buy API

### ISSUE THREAD

**Issue by caohunan** [jaggedsoft/node-binance-api#4]
I think it will be better we have sort of callback function to save the order IDs for the BUY API. 

There is no way to get the order ID for the BUY API. It is a bit scary that we have no way to know whether the BUY action succeeded or not.

**Comment by jaggedsoft** [comment]
Thank you for your report!
This was fixed last week in version 0.2.0
Please navigate to your project directory and issue an `npm update`
Or run the command `npm install node-binance-api --save`
Here are some examples:

#### Limit Buy
```js
var quantity = 5, price = 0.00402030;
binance.buy("BNBETH", quantity, price, {}, function(response) {
	console.log("Limit Buy response", response);
	console.log("order id: " + response.orderId);
});
```

```
Limit Buy response { symbol: 'BNBETH',
  orderId: 4480717,
  clientOrderId: 'te38xGILZUXrPZHnTQPH6h',
  transactTime: 1509049732437,
  price: '0.00402030',
  origQty: '5.00000000',
  executedQty: '5.00000000',
  status: 'FILLED',
  timeInForce: 'GTC',
  type: 'LIMIT',
  side: 'BUY' }
//order id: 4480717
```

#### Market Buy
```js
var quantity = 1;
binance.marketBuy("BNBBTC", quantity, function(response) {
	console.log("Market Buy response", response);
	console.log("order id: " + response.orderId)
	// Now you can limit sell with a stop loss, etc.
});
```

```
Market Buy response {
  symbol: 'BNBETH',
  orderId: 4480553,
  clientOrderId: 'rCGiCG08PGy7AwvbrG5d83',
  transactTime: 1509049376261,
  price: '0.00000000',
  origQty: '1.00000000',
  executedQty: '1.00000000',
  status: 'FILLED',
  timeInForce: 'GTC',
  type: 'MARKET',
  side: 'BUY' }
//order id: 4480553
```

**Comment by caohunan** [comment]
Awesome! Thank you for the quick fix.
